### PR TITLE
Updated MT2MQ.R and MT2MQ.xml

### PR DESCRIPTION
**Pull request by mariecrane**
Added taxize function to fetch NCBI taxon ID for taxonomic mode

### CODE REVIEW

File: tools/mt2mq/MT2MQ.xml [hunk 1]
```diff
@@ -27,6 +28,7 @@
             </param>
             <when value="t">
                 <param name="ontology" type="hidden" value="NA" />
+                <param type="text" name="api_key" value="cab92f653d6d03c4aee4717918a6a925fc08" label="NCBI API key" />
```
Comment (bgruening) [inline tools/mt2mq/MT2MQ.xml:31]: Upps, what is this? We can not include API keys here and in the tool. Please revoke this key if it is a real one - its now leaked :)

Comment (mariecrane) [reply tools/mt2mq/MT2MQ.xml]: You're right! Sorry, big mistake on my part. I just removed it from the code and we're working to replace the old leaked API key with a new one, which will not be shared :)

Comment (mariecrane) [reply tools/mt2mq/MT2MQ.xml]: @bgruening it should finally be ready to merge now! Sorry for the mistakes.

**Comment by bgruening** [comment]
@mariecrane the API key is a general problem. We can not accept credentials in Galaxy tools. Galaxy will store this (reproducibility) in the Galaxy database. In the info-form and so on. So you leak your credentials by sharing histories and so on.

If this is needed I can outline a solution, but its also not great :(

**Comment by mariecrane** [comment]
@bgruening got it! I think it should work fine without the API key so I've just removed that parameter entirely.

File: tools/mt2mq/MT2MQ.xml [hunk 2]
```diff
@@ -1,18 +1,19 @@
...
     <description>Tool to prepare metatranscriptomic outputs from ASaiM for Metaquantome</description>
     <requirements>
         <requirement type="package" version="1.2.1">r-tidyverse</requirement>
+        <requirement type="package" version="0.9.97">r-taxize</requirement>
     </requirements>
     <command detect_errors="exit_code"><![CDATA[
 
 #if $options.mode == "f" or $options.mode == "ft":
-    Rscript '$__tool_directory__/MT2MQ.R' '$options.input_files' '$options.mode' $options.ontology '$mq_output'
+    Rscript '$__tool_directory__/MT2MQ.R' '$options.input_files' '$options.mode' '$options.ontology' '$int_output' '$func_output' '$tax_output'
```
Comment (bernt-matthias) [inline tools/mt2mq/MT2MQ.xml:10]: can you add `--vanilla` ?

Comment (pravs3683) [reply tools/mt2mq/MT2MQ.xml]: Done!! Thanks!!

Comment (pravs3683) [reply tools/mt2mq/MT2MQ.xml]: @bgruening @jj-umn @bernt-matthias @mariecrane 
Hi, Can this update be merged? Required changes were made by @mariecrane and I have added --vanilla as requested. All the tests seem to have passed successfully. Thanks!

**Comment by bgruening** [comment]
Thanks @mariecrane!

**Pull request merged**